recipes/tesseract/all/conanfile.py

```diff
@@ -191,7 +127,7 @@ def package_info(self):
         # TODO: back to global scope once cmake_find_package* generators removed
```

Comment:
Shouldn't the single component be removed along with the rest of Conan v2 cleanup? It was only added to work around cmake_find_package generator limitations. Or should it be kept here and in other recipes for backwards compatibility, in case someone is referring to the component name in a downstream recipe (which is very unlikely, but not impossible)?
Should get rid of this comment either way.

Reply:
Irrespective of ``cmake_find_package`` usage, consumers can reference this as ``self.cpp_info.requires("tesseract::libtesseract")``, so removing this component will break those users. So we are prioritizing stability at the moment and not taking that risk. We will try to implement a escape in the Conan client that allows this, but till then this will be staying.